examples/python/deep_sdf/main.py

```diff
@@ -67,7 +67,7 @@ def get_mesh_format(mesh: Trimesh) -> rr.MeshFormat:
     try:
         return {
             ".glb": rr.MeshFormat.GLB,
-            # ".gltf": MeshFormat.GLTF,
+            # ".gltf": rr.MeshFormat.GLTF,
```

Comment:
this is strange actually. We do support gltf.

Reply:
Until we have a test for it, I wouldn't say we do…